src/fiatconnect/KycLanding.tsx

```diff
@@ -12,7 +14,20 @@ export interface Props {
 }
 
 function KycLanding(_props: Props) {
-  return <SafeAreaView style={styles.container} />
+  const sendKYCSchema = async () => {
+    if (!_props.quote.quoteResponseKycSchema) {
+      throw 'expected KYC schema to send is nullish'
```

Comment:
Can you add a comment saying this should never happen? B/c a user should only be able to reach the kyc flow at all if there is a kyc schema